experimental/dds/tree2/src/test/utils.ts

```diff
@@ -692,6 +692,17 @@ export function createMockUndoRedoManager(): UndoRedoManager<DefaultChangeset, D
 /**
  * Constructs a basic suite of round-trip tests for all versions of a codec family.
  * This helper should generally be wrapped in a `describe` block.
+ *
+ * It is generally not valid to compare the decoded formats with assert.deepEqual,
+ * but since these round trip tests start with the decoded format (not the encoded format),
+ * they require assert.deepEqual is a valid comparison (for example edits containing cursors).
```

Comment:
I'm confused by this last line. Did you mean:
```suggestion
 * assert.deepEqual is a valid comparison (for example edits containing cursors).
```

The point about cursors isn't to me clear either. Are you saying that cursors shouldn't be compared using assert.deepEqual? If so, I think the parenthetical would make more sense after "It is generally not valid to compare the decoded formats with assert.deepEqual".

Reply:
```suggestion
 * they require assert.deepEqual to be a valid comparison.
 * This can be problematic for some cases (for example edits containing cursors).
```